DUI3-DX/Converters/Autocad/Speckle.Converters.AutocadShared/ToHost/Raw/PolycurveToHostPolylineRawConverter.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Linq;
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+
+namespace Speckle.Converters.AutocadShared.ToHost.Raw;
+
+/// <summary>
+/// If polycurve segments consist of only with Line and Arc, we convert it as ADB.Polyline.
+/// </summary>
+public class PolycurveToHostPolylineRawConverter : IRawConversion<SOG.Polycurve, ADB.Polyline>
+{
+  private readonly IConversionContextStack<Document, ADB.UnitsValue> _contextStack;
+  private readonly IRawConversion<SOG.Point, AG.Point3d> _pointConverter;
+
+  public PolycurveToHostPolylineRawConverter(
+    IConversionContextStack<Document, ADB.UnitsValue> contextStack,
+    IRawConversion<SOG.Point, AG.Point3d> pointConverter
+  )
+  {
+    _contextStack = contextStack;
+    _pointConverter = pointConverter;
+  }
+
+  public ADB.Polyline RawConvert(SOG.Polycurve target)
+  {
+    ADB.Polyline polyline = new() { Closed = target.closed };
```

Comment:
Similarly a long one, but probably fine. It's probably that I've a natural aversion to switch statements these days

Reply:
yup, at least we will simply some bulge calculations after `AutocadPolycurve`